docs/api/get_data_from_form_with_guide_spec.rst

```diff
@@ -1,20 +1,35 @@
 get_data_from_form_with_guide_spec function
 ===========================================
 
+Purpose
+-------
+
+This takes a spreadsheet with user entered data and returns the data the user entered.
+
 This is a version of :doc:`get_data_from_form<get_data_from_form>` that should be used where performance is an issue.
 
 :doc:`get_data_from_form<get_data_from_form>` will parse the guide spreadsheet every time it is called.
 If called multiple times, or you need faster action when it is called, use this instead.
 Call :doc:`get_guide_spec<get_guide_spec>` and cache the results. Pass that cached value to this function.
 
+If performance is not an issue, we recommend just using :doc:`get_guide_spec<get_guide_spec>` as that is simpler.
```

Comment:
```suggestion
If performance is not an issue, use :doc:`get_guide_spec<get_guide_spec>`.
```

Reply:
It seems worth saying why; one could read the suggestion and think "surely better performance is always better!?" so explicitly saying it's simpler seems good. Also, by necessity, this page is more complicated, so having a very clear thing going "simpler option over here" seems good.